scripts/qa_crowbarsetup.sh

```diff
@@ -442,9 +442,9 @@ function export_tftpboot_repos_dir()
     if iscloudver 5plus; then
         tftpboot_suse12_dir=/srv/tftpboot/suse-12.0
 
```

Comment:
I think this stays unchanged, as we don't use multiple architectures on cloud 5?